spec/index.html

```diff
@@ -365,6 +380,18 @@ <h3>Terms defined by this specification</h3>
       <dd>A tuple composed of <a>subject</a>, <a>predicate</a>, <a>object</a>, and <a>graph name</a>.
         This is a generalization of an <a>RDF triple</a> along with a <a>graph name</a>.
       </dd>
+      <dt><dfn class="export">canonicalization function</dfn></dt>
+      <dd>A <a>canonicalization function</a> maps <a>RDF datasets</a>
+        into <a data-cite="RDF11-CONCEPTS#dfn-isomorphic">isomorphic datasets</a>.
+        Two datasets produce the same canonical result if and only if they are <a data-cite="RDF11-CONCEPTS#dfn-isomorphic">isomorphic</a>.
+        The <a>RDFC-1.0</a> algorithm implements a <a>canonicalization function</a>.
+        Some datasets may be constructed to prevent this algorithm from
+        terminating (see <a href="#dataset-poisoning" class="sectionRef"></a>),
```

Comment:
Reproducing @dlongley's suggestion which overlapped:

```suggestion
        terminating (see <a href="#dataset-poisoning" class="sectionRef"></a>)
        in a reasonable amount of time,
```